### ISSUE TITLE
In Lighting Call are going but voice is not audible and also not getting any log in Twilio App.

### ISSUE THREAD

**Issue by maggiesxc** [developerforce/open-cti-demo-adapter#13]
Currently calls are going through in Lightning but the voice
 is not audible which works perfectly in classic.

![img4](https://user-images.githubusercontent.com/31212740/29521671-1536b536-86a4-11e7-8d69-031016ddceeb.jpg)

**Comment by vish9845** [comment]
yes its not working the call is redirecting but to the number but voice is not audible..!

**Comment by vish9845** [comment]
Yes call is redirecting to the number but voice is not audible and for incoming call screen is not popingUp for real calls.!

**Comment by manzanofab** [comment]
@maggiesxc did you ever found a solution?

**Comment by manzanofab** [comment]
@maggiesxc I was having same issue, but not logging in twilio means not actually call going out.

I had to change my mobile number to the right format, so in my case (Australia) I changed from 04.... to +614..... and then I was able to make call and properly log in twilio

However, I was not able to talk as when i pick up i only heard back ground music
